### ISSUE TITLE
Frustrating first experience

### ISSUE THREAD

**Issue by runemadsen** [formium/formium#49]
I wanted to take the time to document my first experience using Formium. From the Getting Started guide, it seemed to do exactly what I want: Set up a form for user testing, where I can give my UX designers access to the form builder and I don't need to touch the code. I set up a basic webpack repo, and this is where I encountered my first problem.

**Formium can only be used if you have server-side rendering**

This strikes me as rather odd: I get an API key that needs to be used for getting/submitting forms, but it's also a key for everything else on the platform. This strikes me as an odd choice of architecture. Since I'm building public forms, why do I even need to give an API key? Shouldn't the experience be just like Google Forms where I create a form and everyone can see the fields and submit responses to them? Why do I need an API token for this?

But since this is the case, I decided to whip up a proxy lambda with two routes: One to call `formium.getFormBySlug()` and another to call `formium.submitForm()`. At least my admin API token won't be publicly available in my compiled JS. This led me to another problem:

**The `submitForm` function in the Formium client does not work in Node.js**

I had everything set up and working, but the `submitForm` function is giving me the following error:

```
FormData is not defined
```

So, it relies on the browser's `FormData` object even though I'm not using it, so now that option is out of the picture too.

Just documenting a pretty confusing experience, especially because the documentation clearly states that you can [use the client in the browser without an API token](https://formium.io/docs/api-client#usage) (which returns no fields so it cannot be used) and it's also documented that the client works in Node.js (which is not true).

So, it seems that I'm left with no options than to port my entire codebase to Gatsby or Next.js, which I don't want to do.

**Comment by jaredpalmer** [comment]
@runemadsen thanks a lot for the write up. you don't need your access token to call `submitForm()`, just your project id and your form slug. However, you do need it for fetching your form's schema.

**Comment by holtc** [comment]
Was really excited to use formium as well but also just ran into this issue and wanted to clarify -- there is no way to use formium without server-side rendering, correct?

**Comment by jaredpalmer** [comment]
you can also use the formium-cli and load all of your forms into raw json

**Comment by holtc** [comment]
Gotcha -- I think it would be very helpful to clarify in the documentation that the API client behaves vastly differently depending on having the API token, and thus Formium will not work at all unless server-side rendering is present.

**Comment by skilar** [comment]
I had a similar experience. I created a Lambda function to handle my form submissions, and I got the same error: `FormData is not defined`. I tried adding the `form-data` package to my Lambda layer, but no dice. I even tried converting the form values to a FormData instance in the browser before POSTing them to the Lambda endpoint. Still no dice.

But I don't understand why that error pops up at all, because the documentation says that `submitForm` accepts both FormData and an object. It's also clear [in the code](https://github.com/formium/formium/blob/master/packages/client/src/index.ts#L450). Is there any way to get the client working in a Lambda function?
